bin/setup.js

```diff
@@ -0,0 +1,108 @@
+const spaceImport = require("contentful-import");
+const exportFile = require("../contentful/export.json");
+const inquirer = require("inquirer");
+const chalk = require("chalk");
```

Comment:
`inquirer` and `chalk` aren't installed as deps

Reply:
oh sorry my bad